data/README.md

```diff
@@ -3,21 +3,24 @@
 The `data/` directory contains various files for tracking and submission of
 student contributions to Hacktoberfest 2018/Release 0.2.
 
-**NOTE: submit Pull Requests for any changes you want to make to the files mentioned below.  Every Pull Request must be reviewed and approved by another student before it can be merged.**
+Make sure to run `npm install` before you alter these files so that
+[prettier gets run when you stage your files](https://github.com/humphd/hacktoberfest-at-seneca-2018/pull/11).
+
+**NOTE: submit Pull Requests for any changes you want to make to the files mentioned below. Every Pull Request must be reviewed and approved by another student before it can be merged.**
```

Comment:
Just curious - GitHub has a nice feature that ensures that at least 1 review must be done before a PR can be merged. Enabling it is as easy as visiting [branch protection rules for the repo](https://github.com/humphd/hacktoberfest-at-seneca-2018/settings/branch_protection_rules/), adding `master` as a protected branch and checking `Require pull request reviews before merging`.

The settings screen shout look like:

![screen shot 2018-09-30 at 3 29 32 pm](https://user-images.githubusercontent.com/13009507/46261740-e5ee4800-c4c5-11e8-8646-88ee99b18b90.png)

Reply:
Good idea, I've added a review request protection for `master`.